Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
We can still discuss if we need to change this function from my comment in the third paragraph.